frontend/src/metabase/visualizations/components/skeletons/ChartSkeleton/ChartSkeleton.unit.spec.tsx

```diff
@@ -1,74 +1,89 @@
 import { render, screen } from "@testing-library/react";
-import ChartSkeleton from "./ChartSkeleton";
+import userEvent from "@testing-library/user-event";
+import { CardDisplayType } from "metabase-types/api";
+import ChartSkeleton, { ChartSkeletonProps } from "./ChartSkeleton";
 
-describe("ChartSkeleton", () => {
-  it("should render area", () => {
-    render(<ChartSkeleton display="area" name="Area" />);
-    expect(screen.getByText("Area")).toBeInTheDocument();
-  });
-
-  it("should render bar", () => {
-    render(<ChartSkeleton display="bar" name="Bar" />);
-    expect(screen.getByText("Bar")).toBeInTheDocument();
-  });
+jest.unmock("metabase/components/Popover");
```

Comment:
Should we move that into a `beforeAll`/`beforeEach`?

Reply:
Yeah, that's a decent call, I'll move it to `beforeAll`